app/src/main/res/values/strings.xml

```diff
@@ -151,14 +151,14 @@
     <string name="restart_dialog_restart">Restart Required</string>
 
     <!-- This string is the text displayed in the restart dialog box explaining why the
-         restart is required. '%1$s' will be replaced at runtime with the website's domain name. -->
+         restart is required. '%1$s' will be replaced at runtime with the app's name. -->
     <string name="restart_dialog_text">You must restart %1$s in order to complete changes. Would you like to do that now?</string>
 
     <!-- This string is used to label the button the user presses to restart the
          application in the restart dialog. -->
     <string name="restart_now_dialog_button">Restart Now</string>
 
-    <!-- This string is used to label the button the user presses to restart the postpone the app restart. -->
+    <!-- This string is used to label the button the user presses to postpone the app restart. -->
```

Comment:
Hmmm so maybe it's just me, but I feel like something like: "... the user presses in order to postpone the restart of an app"
may work better?

Reply:
I like that sentence better, yeah let's use your suggestion.

Otherwise I would say "application" or "app" since it's on a phone (also, in case the name changes in the future) but I'm not sure that decision is up to me :)
Maybe that's more for Nancy or Lars to answer?

Reply:
@Delphine Fixed that